po/ru_RU.po

```diff
@@ -0,0 +1,264 @@
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:57
+msgid "Performance and stability"
+msgstr "Производительность и стабильность"
+
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:11
+msgid "Colour Scheme Preference"
+msgstr "Предпочтение цветовой схемы"
+
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:12
+msgid ""
+"Whether to follow the system colour scheme or force either light or dark "
+"mode."
+msgstr ""
+"Следует ли следовать системной цветовой схеме или использовать светлый или "
+"темный режим."
+
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/exm-application.c:112
+msgid "translator-credits"
+msgstr "translator-credits Danik2343 (Russian)"
+
+#: src/exm-application.c:113
+msgid ""
+"A very simple tool for browsing, downloading, and managing GNOME shell "
+"extensions."
+msgstr ""
+"Очень простой инструмент для просмотра, загрузки и управления расширениями "
+"оболочки GNOME."
+
+#: src/exm-application.c:119
+msgid "Project Homepage"
+msgstr "Домашняя страница проекта"
+
+#: src/exm-browse-page.blp:7
+msgid "Search for extensions"
+msgstr "Поиск расширений"
+
+#: src/exm-browse-page.blp:8
+msgid ""
+"Enter a keyword to search 'extensions.gnome.org' for GNOME Shell Extensions."
+msgstr ""
+"Введите ключевое слово для поиска в 'extensions.gnome.org' расширений GNOME "
+"Shell."
+
+#: src/exm-browse-page.blp:22
+msgid "e.g. \"Blur my Shell\""
+msgstr "например \"Blur my Shell\""
+
+#. Translators: dropdown items for sorting search results
+#: src/exm-browse-page.blp:33
+msgid "Popularity"
+msgstr "Популярность"
+
+#: src/exm-browse-page.blp:33
+msgid "Downloads"
+msgstr "Загрузки"
+
+#: src/exm-browse-page.blp:33
+msgid "Recent"
+msgstr "Недавние"
+
+#: src/exm-browse-page.blp:33
+msgid "Name"
+msgstr "Название"
+
+#: src/exm-browse-page.blp:60
+msgid "No Results Found"
+msgstr "Результаты не найдены"
+
+#: src/exm-detail-view.blp:35 src/exm-detail-view.c:287
+msgid "An Error Occurred"
+msgstr "Произошла ошибка"
+
+#: src/exm-detail-view.blp:95
+msgid "Description"
+msgstr "Описание"
+
+#: src/exm-detail-view.blp:113
+msgid "Supported Versions"
+msgstr "Поддерживаемые версии"
+
+#: src/exm-detail-view.blp:129
+msgid "Links"
+msgstr "Ссылки"
+
+#: src/exm-detail-view.blp:139
+msgid "View on Extensions"
+msgstr "Посмотреть в Extensions"
+
+#: src/exm-detail-view.c:120
+msgid "This extension is incompatible with your current version of GNOME."
+msgstr "Это расширение несовместимо с вашей текущей версией GNOME."
+
+#: src/exm-detail-view.c:129 src/exm-search-row.blp:75
+msgid "Install"
+msgstr "Установить"
+
+#: src/exm-detail-view.c:134 src/exm-search-row.c:153 src/exm-window.blp:42
+msgid "Installed"
+msgstr "Установлено"
+
+#: src/exm-detail-view.c:138 src/exm-search-row.c:159
+msgid "Unsupported"
+msgstr "Не поддерживается"
+
+#. Translators: Use unicode ellipsis '…' rather than three dots '...'
+#: src/exm-detail-view.c:303
+msgid "Loading…"
+msgstr "Загрузка..."
```

Comment:
Please use the unicode '…' character instead of three dots. Just for consistency with the other translations :)

```suggestion
msgstr "Загрузка…"
```